src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,139 @@
 package mate.academy.lib;
 
+import java.io.File;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 public class Injector {
     private static final Injector injector = new Injector();
+    private static final Map<Class<?>, Class<?>> interfacesImplementations;
```

Comment:
From my point of view, we need to follow KISS principles and use just "Map.of(...)" for interface implementations.